packages/vue/src/components/search/DataSearch.jsx

```diff
@@ -67,9 +69,11 @@ const DataSearch = {
 		const props = this.$props;
 		this.__state = {
 			currentValue: '',
+			selectedTags: [],
 			isOpen: false,
 			normalizedSuggestions: [],
 			isPending: false,
+			isTagsMode: props.mode === SEARCH_COMPONENTS_MODES.TAG,
```

Comment:
`mode` can not be dynamic, hence it should not be defined in `data`. Define it as a class property